assimilation_code/programs/readme.rst

```diff
@@ -0,0 +1,206 @@
+Programs
+========
+
+This list of programs is separated (imprecisely) into groups which have similar functionality.
+Within each group they are sorted (approximately) by the order
+in which they might be used andor by their usefulness.
+
+">>" marks the most commonly used and up to date programs.
+
+Setting Up Experiments
+-----------------------------------
+ 
+`>> model_mod_check/model_mod_check.f90 <model_mod_check/model_mod_check.html>`_
+   "Hackable" program to test some of the more fundamental routines in any ``model_mod``, 
+   especially a new one.
+
+`>> preprocess/preprocess.f90 <preprocess/preprocess.html>`_
+   Program to insert observation specific code into DART before filter or perfect_model_obs is compiled.
+ 
+`>> fill_inflation_restart/ fill_inflation_restart.f90* <fill_inflation_restart/fill_inflation_restart.html>`_
+   Create inflation restart files with constant values taken from ``filter_nml``.
```

Comment:
i'm pretty sure fill_inflation_restart has its own namelist and doesn't use filters.

Reply:
You're right.  I'll fix it.